code/modules/mining/abandoned_crates.dm

```diff
@@ -24,42 +25,41 @@
 	switch(loot)
 		if(1 to 5) //5% chance
 			new /obj/item/reagent_containers/food/drinks/bottle/rum(src)
-			new /obj/item/reagent_containers/food/snacks/grown/ambrosia/deus(src)
+			new /obj/item/reagent_containers/food/drinks/bottle/whiskey(src)
 			new /obj/item/reagent_containers/food/drinks/bottle/whiskey(src)
 			new /obj/item/lighter(src)
+			new /obj/item/reagent_containers/food/drinks/bottle/absinthe/premium(src)
+			new /obj/item/clothing/mask/cigarette/rollie(src)
+			new /obj/item/clothing/mask/cigarette/rollie(src)
+			new /obj/item/clothing/mask/cigarette/rollie(src)
 		if(6 to 10)
-			new /obj/item/bedsheet(src)
-			new /obj/item/kitchen/knife(src)
-			new /obj/item/wirecutters(src)
-			new /obj/item/screwdriver(src)
-			new /obj/item/weldingtool(src)
-			new /obj/item/hatchet(src)
-			new /obj/item/crowbar(src)
+			new /obj/item/melee/skateboard/pro(src)
 		if(11 to 15)
-			new /obj/item/reagent_containers/glass/beaker/bluespace(src)
+			new /mob/living/simple_animal/bot/honkbot(src)
 		if(16 to 20)
 			new /obj/item/stack/ore/diamond(src, 10)
 		if(21 to 25)
 			for(var/i in 1 to 5)
 				new /obj/item/poster/random_contraband(src)
 		if(26 to 30)
-			for(var/i in 1 to 3)
-				new /obj/item/reagent_containers/glass/beaker/noreact(src)
+			new /obj/item/vending_refill/sovietsoda(src)
+			new /obj/item/circuitboard/machine/vending(src)
 		if(31 to 35)
 			new /obj/item/seeds/firelemon(src)
 		if(36 to 40)
-			new /obj/item/melee/baton(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
 		if(41 to 45)
-			new /obj/item/clothing/under/shorts/red(src)
-			new /obj/item/clothing/under/shorts/blue(src)
+			new /obj/item/reagent_containers/food/snacks/burger/roburger(src)
```

Comment:
this burger actually has robot transmutation which basically is a disease that force-borgs you.